src/vm/checker/typecheck/tests/contracts.rs

```diff
@@ -42,53 +39,55 @@ fn test_names_tokens_contracts() {
          (define-map preorder-map
            ((name-hash (buff 20)))
            ((buyer principal) (paid int)))
-
-         (define-read-only (get-balance)
-           ;; adding zero ensures that contract-call! is correctly
-           ;;   returning an integer type here. 
-           (+ 0 (contract-call! tokens get-balance tx-sender)))
          
          (define-public (preorder 
                         (name-hash (buff 20))
                         (name-price int))
-           (if (contract-call! tokens token-transfer
-                 burn-address name-price)
-               (insert-entry! preorder-map
-                 (tuple (name-hash name-hash))
-                 (tuple (paid name-price)
-                        (buyer tx-sender)))
-               'false))
+           (if (print (is-ok? (contract-call! tokens token-transfer
```

Comment:
Does `(print` eval to a `bool` now?

Reply:
`(print ...)` always evals to it's argument -- so if I call

`(print (+ 1 2))`

it evals to `3`

Reply:
Ah okay.